contrib_bots/lib/githubcomment.py

```diff
@@ -0,0 +1,148 @@
+                     json.dumps(comment))
+
+    return r.status_code
+
+
+def get_values_message(original_content, client, original_sender):
+    # gets rid of whitespace around the edges, so that they aren't a problem in the future
+    message_content = original_content.strip()
+
+    # splits message by '/' which will work if the information was entered correctly
+    message_content = message_content.split('/')
+
+    try:
+        # this will work if the information was entered correctly
+        user = message_content[1]
+        repo_owner = message_content[2]
+        repo = message_content[3]
+        issue = message_content[4]
+        comment_body = message_content[5]
+
+        return {'user': user, 'repo_owner': repo_owner, 'repo': repo, 'issue': issue, 'comment_body': comment_body}
+    except IndexError:
+        raise InputError
+
+
+def handle_input(client, original_content, original_sender):
+
+    try:
+
+        params = get_values_message(original_content, client, original_sender)
+
+        status_code = send_to_github(params['user'], params['repo_owner'], params['repo'],
+                                     params['issue'], params['comment_body'])
+
+        if status_code == 201:
+            # sending info to github was successful!
+            reply_message = "You commented on issue number " + params['issue'] + " under " + \
+                            params['repo_owner'] + "'s repository " + params['repo'] + "!"
+
+            send_message(client, reply_message, original_sender)
+
+        elif status_code == 404:
+            # this error could be from an error with the OAuth token
+            reply_message = "Error code: " + str(status_code) + " :( There was a problem commenting on issue number " \
+                            + params['issue'] + " under " + \
+                            params['repo_owner'] + "'s repository " + params['repo'] + \
+                            ". Do you have the right OAuth token?"
+
+            send_message(client, reply_message, original_sender)
+
+        else:
+            # sending info to github did not work
+            reply_message = "Error code: " + str(status_code) +\
+                            " :( There was a problem commenting on issue number " \
+                            + params['issue'] + " under " + \
+                            params['repo_owner'] + "'s repository " + params['repo'] + \
+                            ". Did you enter the information in the correct format?"
+
+            send_message(client, reply_message, original_sender)
+
+    except InputError:
+            message = "It doesn't look like the information was entered in the correct format." \
+                      " Did you input it like this? " \
+                      "'/<username>/<repository_owner>/<repository>/<issue_number>/<your_comment>'."
+            send_message(client, message, original_sender)
+            print('there was an error with the information you entered')
```

Comment:
@showell I think I've heard about it, but I've never used it. Would that be better here?